app/forms/webauthn_setup_form.rb

```diff
@@ -71,8 +87,12 @@ def name_is_unique
         count
       @name = "#{@name} (#{num_existing_devices})"
     else
-      errors.add :name, I18n.t('errors.webauthn_setup.unique_name'), type: :unique_name
-      @name_taken = true
+      name_error = if platform_authenticator?
+                     I18n.t('errors.webauthn_platform_setup.unique_name')
+      else
+        I18n.t('errors.webauthn_setup.unique_name')
+      end
```

Comment:
i think this is splitting two indentation styles, 
```suggestion
      name_error = if platform_authenticator?
                     I18n.t('errors.webauthn_platform_setup.unique_name')
                   else
                     I18n.t('errors.webauthn_setup.unique_name')
                   end
```